v0/util/constant.js

```diff
@@ -45,7 +50,12 @@ const TRANSFORMER_METRIC = {
         BAD_EVENT: "badEvent",
         BAD_PARAM: "badParam",
         INSTRUMENTATION: "instrumentation",
-        CONFIGURATION: "configuration"
+        CONFIGURATION: "configuration",
+        /**
+         * Basically this means that the error is an expected error(thrown during transformation)
+         * This meta will be used for CustomError thrown during transformations
+         */
+        CLIENT_ERROR: "clientError"
```

Comment:
We have to probably rename this something like THROWN error or something. Basic indication is that rudderstack knows about this error(you can say this is expected error since there was something wrong in payload)

Reply:
**Note:** This is open to suggestion

Reply:
IMO `CLIENT_ERROR` is ambiguous and misleading. I'd suggest `HANDLED`.

Reply:
How about we rename it to `EXPECTED`. It indicates that an error was thrown but a specific `meta` tag was not added to the thrown error.

Reply:
The changes requested are done